spark/src/main/java/org/apache/iceberg/actions/BaseSnapshotUpdateSparkAction.java

```diff
@@ -23,7 +23,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.apache.spark.sql.SparkSession;
 
-abstract class BaseSnapshotUpdateSparkAction<ThisT, R>
+public abstract class BaseSnapshotUpdateSparkAction<ThisT, R>
```

Comment:
This had to be made public temporarily. I'll revert this change in a follow-up commit.